src/main/java/dev/hephaestus/glowcase/Glowcase.java

```diff
@@ -41,22 +40,24 @@
 public class Glowcase implements ModInitializer {
 	public static final String MODID = "glowcase";
 
+	public static GlowcaseCommonProxy proxy = new GlowcaseCommonProxy(); //Overridden in GlowcaseClient
+	
 	public static final TagKey<Item> ITEM_TAG = TagKey.of(RegistryKeys.ITEM, id("items"));
 
 
-	public static final Block HYPERLINK_BLOCK = Registry.register(Registries.BLOCK, id("hyperlink_block"), new HyperlinkBlock());
```

Comment:
These field-type changes were made so that other areas of the code can call the new `GlowcaseBlock#canEditGlowcase` method.